test.js

```diff
@@ -0,0 +1,59 @@
+const assert = require('assert');
+
+Object.freeze(assert);
+const sumOfOther = require('./src/sumOfOther.js');
+const make = require('./src/make.js');
+const recursion = require('./src/recursion.js');
+
+const sum = (a, b) => a + b;
+const pow = (a, b) => a ** b;
+const max = (a, b) => Math.max(a, b);
+
+const treeOne = { value: 100, left: { value: 90, left: { value: 70 }, right: { value: 99 } }, right: { value: 120, left: { value: 110 }, right: { value: 130 } } };
+const treeTwo = { value: 150, left: { value: 120, left: { value: 100 } }, right: { value: 170, right: { value: 200 } } };
+const treeThree = { value: 200, left: { value: 100 }, right: { value: 300 } };
+
+describe('sumOfOther', () => {
+  it('1', () => {
+    const arr = sumOfOther([2, 3, 4, 1]);
+    assert.deepEqual(arr, [8, 7, 6, 9]);
+  });
+  it('2', () => {
+    const arr = sumOfOther([163, 107, 52, 159, 76, 96, 96, 104, 94, 44, 26, 26, 110, 142, 182, 142, 113, 75, 62, 57]);
```

Comment:
Here and below you've violated the eslint rule about _maximum line length_.
I assume you just haven't checked whether your **test.js** file passes the eslint rules.

~**TL:DR:** Use ESLint here also, please :)
Unit-test spec files are actually quite important in software development processes. And without any doubts obligatory on any more or less big project. Devs spend quite a lot of type on analyzing, writing and maintaining of units-tests which means that it recommended to treat them the same way as you treat your code base.